packages/test/test-end-to-end-tests/src/test/gc/gcDatastoreAliased.spec.ts

```diff
@@ -18,27 +18,25 @@ import { getGCStateFromSummary } from "./gcTestSummaryUtils";
  */
 describeFullCompat("GC Data Store Aliased", (getTestObjectProvider) => {
     let provider: ITestObjectProvider;
-    let container1: IContainer;
-    let container2: IContainer;
-    let mainDataStore1: ITestDataObject;
-    let mainDataStore2: ITestDataObject;
+
+    beforeEach(async () => {
+        provider = getTestObjectProvider({ syncSummarizer: true });
+    });
 
     async function waitForSummary(container: IContainer) {
         const dataStore = await requestFluidObject<ITestDataObject>(container, "default");
         return (dataStore._context.containerRuntime as ContainerRuntime).summarize({ runGC: true, trackState: false });
     }
 
-    beforeEach(async () => {
-        provider = getTestObjectProvider({ syncSummarizer: true });
-        container1 = await provider.makeTestContainer(defaultGCConfig);
-        container2 = await provider.loadTestContainer(defaultGCConfig);
-        mainDataStore1 = await requestFluidObject<ITestDataObject>(container1, "default");
-        mainDataStore2 = await requestFluidObject<ITestDataObject>(container2, "default");
+    // Note: this is an edge case.
+    it("GC aliased datastore is still aliased when initially summarized as non-aliased datastore", async () => {
```

Comment:
This title is not very clear and it took me a while to understand what this test is doing from it :-).